src/client/components/equalizer/canvasEqualizer/canvasEqualizer.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Enzyme, {
+  shallow,
+} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import CanvasEqualizer from './canvasEqualizer';
+
+Enzyme.configure({
```

Comment:
Don’t you
Add this config, to config file?

Reply:
ok